third_party/system_libraries.BUILD

```diff
@@ -21,14 +21,16 @@ config_setting(
 cc_library(
     name = "folly",
     srcs = select({
-        ":use_folly_so": ["usr/local/lib/libfolly.so"],
+        ":use_folly_so": [],
         "//conditions:default": [
             "usr/local/lib/libfolly.a",
             "usr/local/lib/libfmt.a",
         ],
     }),
     linkopts = select({
         ":use_folly_so": [
+            "-L/usr/local/lib",
```

Comment:
Can you explain why this was necessary? Or was it trial and error? I would have thought my `ldconfig` would fix this.. maybe.

Reply:
Trial and error. I ran ldconfig but that didn't seem to help

Reply:
Can you show me the error you get when you lack -L/usr/locallib?

Reply:
so just checked that the devcontainer works without "-L/usr..."
in the Magma VM, let me try agian..

Reply:
scratched that this is the error even on the devcontainer: `/usr/bin/ld.gold: error: cannot find -lfolly`